.gitignore

```diff
@@ -1,8 +1,18 @@
 # 內含dsn隱密資訊
 # 格式：DBname    UserName   PassWord     HostName     PortNumber 
 dsn_config.txt
+test_dsn_config.txt
 # 測試檔案
 test.ts
 ```
 # unecessary files
-views
\ No newline at end of file
+views
+dist/
+package-lock.json
+pkg/mod/
+node_modules/
+FakeLaneInfo.json
+go.mod
```

Comment:
`go.mod` describes your dependencies, and `go.sum` is a lock file. You should leave both of them on git.